test/test_integration_ractor.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require 'helper'
+require 'fileutils'
+
+class TC_Integration_Ractor < SQLite3::TestCase
+  STRESS_DB_NAME = "stress.db"
+
+  def setup
+    teardown
+  end
+
+  def teardown
+    FileUtils.rm_rf(Dir.glob "#{STRESS_DB_NAME}*")
+  end
+
+  def test_ractor_share_database
+    skip('Requires Ruby with Ractors') if RUBY_VERSION < '3.2.0'
```

Comment:
Can I ask that you add a predicate method on the `SQLite3` module, similar to `.threadsafe` and `.sqlcipher?` that can be used self-describe and to skip tests when necessary? It's much more descriptive than a version number check.

Reply:
(And print out the value of that predicate in `test/helper.rb` where we print the other info.)